app/views/flush.py

```diff
@@ -0,0 +1,74 @@
+from app.globals import get_answer_store, get_metadata, get_questionnaire_store
+from app.submitter.submitter import SubmitterFactory
+from app.submitter.converter import convert_answers
+from app.utilities.schema import get_schema
+
+from app.authentication.flush_permission_denied import FlushPermissionDenied
+from app.authentication.jwt_decoder import JWTDecryptor
+from app.authentication.map_role_to_permissions import map_role_to_permissions
+from app.authentication.no_survey_data_to_flush import NoSurveyDataToFlush
+from app.authentication.no_token_exception import NoTokenException
+from app.authentication.user import User
+from app.authentication.user_id_generator import UserIDGenerator
+
+from app.questionnaire.path_finder import PathFinder
+
+from flask import Blueprint, Response, request, session
+
+flush_blueprint = Blueprint('flush', __name__)
+
+
+@flush_blueprint.route('/flush', methods=['GET'])
+def flush_data():
+
+    if session:
+        session.clear()
```

Comment:
Inadvertently it already does,as the tests in the setup populate the data, meaning there is a current session, which has to be clear. Its how I noticed the problem originally.